.github/workflows/eden_gcp.yml

```diff
@@ -82,9 +82,12 @@ jobs:
       - name: Collect logs
         if: ${{ always() }}
         run: |
-          ./eden log --format json > trace.log
-          ./eden info > info.log
-          ./eden utils gcp vm log --vm-name eve-eden-actions-${{ matrix.hv }}-${{github.run_number}} -k "$GOOGLE_APPLICATION_CREDENTIALS" > console.log
+          ./eden log --format json > trace.log || echo "no log"
```

Comment:
why do we need it?

Reply:
In case of no-onboarded device this command will fail. But we want to see devise console logs.